src/prefect/orion/schemas/actions.py

```diff
@@ -31,159 +35,133 @@ def validate_block_document_name(value):
     return value
 
 
-class FlowCreate(
-    schemas.core.Flow.subclass(
-        name="FlowCreate",
-        include_fields=["name", "tags"],
-    )
-):
+@copy_model_fields
+class FlowCreate(PrefectBaseModel):
     """Data used by the Orion API to create a flow."""
 
+    name: str = FieldFrom(schemas.core.Flow)
+    tags: List[str] = FieldFrom(schemas.core.Flow)
+
     class Config:
         extra = "forbid"
 
 
-class FlowUpdate(
-    schemas.core.Flow.subclass(name="FlowUpdate", include_fields=["tags"])
-):
+@copy_model_fields
+class FlowUpdate(PrefectBaseModel):
     """Data used by the Orion API to update a flow."""
 
+    tags: List[str] = FieldFrom(schemas.core.Flow)
+
     class Config:
         extra = "forbid"
 
 
-class DeploymentCreate(
-    schemas.core.Deployment.subclass(
-        name="DeploymentCreate",
-        include_fields=[
-            "name",
-            "version",
-            "flow_id",
-            "schedule",
-            "is_schedule_active",
-            "work_queue_name",
-            "description",
-            "tags",
-            "parameters",
-            "manifest_path",
-            "parameter_openapi_schema",
-            "storage_document_id",
-            "path",
-            "entrypoint",
-            "infrastructure_document_id",
-            "infra_overrides",
-        ],
-    )
-):
+@copy_model_fields
+class DeploymentCreate(PrefectBaseModel):
     """Data used by the Orion API to create a deployment."""
 
+    name: str = FieldFrom(schemas.core.Deployment)
```

Comment:
I find there's a lot of visual noise from using the extra module `schemas.` in every field definition.  What do you think about `from prefect.orion.schemas import core, states` and having these all be `core.Deployment`, `core.FlowRun`, `states.State`, etc?

Reply:
I'd be willing to do that or just import the objects directly.

Reply:
Thoughts about shortening these?  It could also be done in a follow-up PR

Reply:
I didn't make this change because we're using `schemas.` frequently in this library — it's kind of our standard pattern. I'd like to address this later perhaps more holistically but for now I want to be consistent.